### ISSUE TITLE
Open Discussion for a Dark Theme edition

### ISSUE THREAD

**Issue by DirtDiglett** [DirtDiglett/Pressure2#42]
# This is for an open discussion regarding a Dark Themed edition of Pressure².

I've had some time to think about it, I have read every single comment from every person that has provided their thoughts or opinions - no matter how antagonistic or douchey - and had a bit of self reflection. I have come to the conclusion that I am not going to commit to one way or the other just yet but first have an open discussion with you guys - the fans. 
### Like any good marriage, it's about compromise.

I have read a lot of comments on the matter, most of them don't really specify any reasoning but those that do have said their main concern is that the light theme is quite harsh on their eyes. My go-to response thus far has been "Well, the majority of applications and websites are white or light themed, why's this any different?" however, I have reasoned that perhaps the brightness could be an issue when in-game and using the overlay, there would potentially be a harsh transition from any dark games to a bright interface.

Perhaps we can meet halfway. **What if I was to change Pressure² so that when in-game and using the overlay it utilized a dark theme?** This would have the added benefit of also resolving issues with text looking a bit shitty in the overlay (Thanks Valve, for using crummy text rendering in the overlay!)

I'm going to experiment and come up with a way to make the in-overlay dark theme functional, and if this is well received it'll become part of the default Pressure².

---

The floor is open, what're your thoughts? Please though, don't just say shit like "Dark theme or bust", actually elaborate on the why, provide some logical arguments for or against.

**Comment by Bromethius** [comment]
Or you know, you could do like the Air interface people did and give people the option of either the white or black color schemes.

**Comment by DarkThemePlease** [comment]
Most well developed things will include both options, not just when discussing dark themes, but any contrasting features. The existence and popularity of dark themes, and the fact that it has grabbed your attention is proof that it deserves implementation.

**Comment by techNeffect** [comment]
I just do not like bright applications. I like nice, subdued, dark themes on all sites and programs that I have the option of making them so. As for the compromise, to be honest I'd just use a different skin. That bright white UI is like nails in my eyes, especially at night. Us blue-eyed folk tend to be photosensitive (there, medical reasons, heh).

I think a better compromise would be to allow for a completely dark theme with a setting or an ini tweak.

**Comment by leeroy1** [comment]
Users who use Steam late at night prefer a darker theme, as it isn't so bright to look at. I prefer a darker skin to view not only because of this on the main client, but also in the chat. It is just too weird to look at, it strains the eyes less as well. This is why so many people loved Pressure. People who want a dark skin **do not** dislike the current design, they just dislike the amount of white in the program. I for one love the design, but the strain on my eyes as well as the difference between the dark and light versions (not being used to the light version) was just too much for me to convert over, so I went back to the old Pressure because of this.

**Comment by SlyAceZeta** [comment]
> What if I was to change Pressure² so that when in-game and using the overlay it utilized a dark theme?

That's not what the request is. All of us asking for a dark version of Pressure² are asking for the entirety of Pressure² to be dark-themed, not just while in-game or using the overlay. You're right, most applications and websites do not support or feature dark themes, however Steam users are a unique bunch in that a lot of us do most of our gaming late at night (after school/work). Having a bright white theme this late at night is not a good thing, and it's why most Steam themes are dark by default.

@leeroy1 summed it up perfectly:

> People who want a dark skin do not dislike the current design, they just dislike the amount of white in the program.

**Comment by Kobi-Blade** [comment]
The time used on developing a Dark Skin, could be better used on improving and fixing Pressure².

You already promised to optimize the older Pressure Skin, and that should be enough for anyone interested in a Dark Theme.

If people keep pushing for a Dark Theme on Pressure², the older variant (Pressure) should be dropped.

**Comment by Hatomike** [comment]
For me, a dark theme is just a preference. My whole windows use a custom dark theme and that why I wanted my steam dark too. And the steam website is dark too, so a light client that is used to navigate on a dark website is strange for some peoples.

Like a lots say, you should keep the light theme by default and add a dark theme in config.ini

**Comment by SlyAceZeta** [comment]
@JoseSnake

> The time used on developing a Dark Skin, could be better used on improving and fixing Pressure².

A dark variant of Pressure² isn't a waste of time or resources, it's something a lot of people would utilize. It's not like it needs to be a completely separate theme - couldn't it just be included in some sort of config.ini or settings.ini to change ourselves?

**Comment by DirtDiglett** [comment]
I'll clarify up front, a simple config.ini to enable a dark theme is not going to be possible - it would **have** to be a completely seperate version. If a dark theme was developed it would require that a huge chunk of the images are also adjusted appropriately, that a buttload of the styling changed. It's not just invert all the colors have a good day, there'd be significant time and energy devoted to doing the overhaul.

**Comment by Hatomike** [comment]
I was thinking you could do something like this: http://www.metroforsteam.com/
In this skin, you can edit background, color, overlay color, etc... In the config.ini
I just posted this if you need an example.

**Comment by Kobi-Blade** [comment]
@Hatomike Uses same styles, just different colours (and darker version has different styles on extras).

**Comment by Hatomike** [comment]
@lKobi My bad, didn't know.

**Comment by BusterNeece** [comment]
Absolutely agree with the requests for the option to have a dark theme available that applies across the entire UI. Yes, a number of business-grade applications don't offer this, but when you look at software like Discord and even unskinned Steam itself, it's clear that they're designed from the ground up for people playing games in darker environments, where the eye strain of a UI isn't necessary or helpful.

I would also have no objection to the other proposal, which is that Pressure 1 be dropped entirely in favor of a darker theme for Pressure 2. Clearly, a huge amount of work has gone into Pressure 2's development, and if that same work could be applied to a dark theme, I know it would really make my day.

**Comment by Limboman** [comment]
First let me say that I love the design and layout of the theme, it really is beautiful. That being said I would much prefer a darker option for the entire theme, I know that you want a logical reason as to why it should be created and what other people have said is true, at night a darker theme is better on the eyes but I can't honestly provide a logical reason as to why I personally would like a darker version apart from the fact that I would find it far more aesthetically pleasing. However I can totally appreciate that "because some guy finds it far more aesthetically pleasing" is probably not a good enough reason to spend a large amount of time creating a new version of the skin that being said I would urge you to consider it as it seems quite a few people would appreciate it (including me).

**Comment by SHADOW-XIII** [comment]
I think there are few issues how people perceived P2. 
First and foremost it was nearly opposite of P1, while P1 was nicely balanced dark, nicely altering existing Steam the same time, the changes weren't too much. P2 dropped on people without anyone knowing what to expect (to my knowledge no screenshots were trailed), most likely expecting P1 in some kind of more advanced form. 
Then P2 turned out nothing like P1.

I love P1 myself, I think its amazing tweak versus plain Steam but P2 is just not for me, mainly because the pure white is hard for my eyes, probably too much contrast with black?.
I have applied P2 and the whiteness hit me instantly, wasn't easy on the eyes - maybe after few more minutes or hours looking at it they would use to. I imagine you're fine as you worked on it for some time but other people were greeted with sudden white colour hitting their eyes where dark/black used to be. Imagine something you see in real life every single, something you see up close and dark (and often surrounding might be dark) and suddenly gives pure bright white colour. It might be tiresome for eyes to get used to for brain to adjust to it.

On the other hand I love P1 so much, to be honest I am not sure I would switch to P2 if it had dark mode either.

If you are going to compromise (and regardless it's appreciated you are considering it), I would suggest doing a mockup screen or two and showing it to people. 
I think it's better to let people know what to expect or receive feedback during the work on something than at the end of it.
Maybe some kind of config where you can specify colour(s) or greysacale amount (ie 0-255, 0=black, 255=white or vice versa)

One day I would love if my Steam could look like [this](http://rendom.net/steam3/index.html) but that's a fairy tale.

Regardless of what you choose and how you do it, I am grateful on your work with Pressure

**Comment by sebj** [comment]
I appreciate the work you've done on Pressure 1 (which I currently use) and Pressure 2, but without going all _"dark theme or bust"_ as you've said, I find it hard to use P2 without it being significantly darker/less bright.

[F.lux](https://justgetflux.com/) tints and dims my screen at night on Windows and OS X, and [Twilight](https://play.google.com/store/apps/details?id=com.urbandroid.lux&hl=en_GB) does the same on Android. It also switches to a system dark node for my menubar and dock on OS X, which some 3rd party apps also support. Android N is rumoured to have a system dark mode; many apps (Reddit apps) support manual/auto switching; [iOS 9.3](http://www.apple.com/ios/preview/) includes 'Night Shift'. See also [YouTube Gaming ](https://gaming.youtube.com/) etc.

Dark modes (and F.lux etc) exist to reduce bright light (and blue light) on our eyes at night, so for a gaming-related application it'd make sense, and is probably why Steam's default UI, and loads of other skins have a dark theme (as well as the connotations of darker colours). If anything, I'm curious why you chose to make this a light/white-based skin (?)

I'm also definitely in favour of the previously mentioned idea  (@SlvrEagle23)  to stop supporting/working on Pressure 1 and replace it with a dark version of Pressure 2.

**Comment by sejbr** [comment]
@SHADOW-XIII Doing a screenshot to show how the skin would look on the darktheme is pointless, there's already an edit changing some stuff to make it dark http://steamcommunity.com/groups/pressureskin/discussions/0/405692758703630874/ not perfect but works and you can see how the skin looks being dark and to be honest It looks great. Would be even better done officially since there are some places that need work like images still being the white ones. I'd also assume there would be problems with updates since they could be delayed/bugged/don't show up at all after some time.

As to the official dark version and my reasoning, first and foremost I do sometimes work with random server owners on different timezones and I tend to open a whole bunch of chat windows on the second screen. Having a whole bright white screen shining it's ominous blinding light upon thee at 3/4AM is not at any case a pleasant experience. Thus personally for me making only the overlay dark would not be sufficient, welcome nonetheless. I'll not bring up the case of some people liking darker ui's more than white ones, since I guess the reason for current theme being white is that the creator enjoyed it the most. Not sure if there's a saying like that in english but in polish we tend to sometimes say "Opinions are like asses - everyone has one" and arguing about the theme would be pointless. If you don't want to do the dark theme It's up to you, pushing creative people to do something for the masses despite their vision is horrible. I think many people will at least be happy if the overlay is dark and I'll hope that someone will fork it and do and maintain a fullon dark theme. I'd do it but I suck at design.

As some people said, for white sites I use browser extensions and most text editors and IDE's have dark colorschemes and themes so you can go full darkmode if you wish with most software.

**Comment by willyb321** [comment]
@sejbr Damn, that actually looks really nice!



-- Will

**Comment by jessb321** [comment]
@sejbr yeah, that looks really nice. I personally (and a lot of others it seems) would like both the overlay and the normal UI to have a dark mode myself, but atm I'm just using Pressure1, as the white is really too bright for my eyes most of the time. I do understand that steam skins aren't as easy as flipping a switch, but I would agree to drop p1 and just focus on p2 as I really do like the design more than p1, just the white is really too bright for me.

**Comment by Kierne** [comment]
The light version of P2 is tiring on my eyes because the text doesn't stand out well in all that open white space. On this page for example, the text is confined to a narrow strip in the center and posts are squared off with boxes, so it's easy for the eye to follow. With light-P2, I find myself consciously having to direct my eyes around the Steam window to look for text before I can actually read it. With dark-P1, my eyes are drawn to the lighter areas of text because they stand out better.

**Comment by notNSANE** [comment]
Here there are my thoughts as an UX "aware" designer.

When you think about UX, you think about context. Real life context. I'm willing to bet an arm that most people that play games, do so at night, and for several hours. That's when this "dark theme" request comes from: the actual playing environment is dark, it makes sense.

I like your light approach, I use several light apps myself, but yours feel like a chimera. It's light, yes, but there are still many darker areas and stuff that feels mashed against each other. P2 Takes Material design and applies to Windows/OSX/Linux... well... Material design is for Android only, and it makes sense there. If you take a look at Metro (ewww), Universal Apps are mostly "light", but the "container" is actually dark (Taskbar, Start Menu, etc.)

I'd say, if you're going to keep a light theme, keep it **everywhere**. "Steal" Metro as inspiration and reduce black bars and backgrounds. 

Either that or scratch it and start a dark theme. Personally, I'm still using Pressure 1 and will continue to do so. You said that you don't want to commit, but you should. P2 is actually a chimera, please don't make it even more. Either commit to a full light, or full dark theme. People with obviously choose the latter, but I might use a full light theme, I just don't like where's at now.

That being said, it's an awesome piece of inside work! Congrats for that.

**Comment by vinniegnsb** [comment]
I like the theme a lot. BUT I do not like how the text in the steam overlay, like when looking at the chat window, has not enough contrast between itself and the background. The text should be a darker black, instead of this faded out stuff. The way it is at now my eyes feel like they might die. Also, I have no problem with the in-game overlay, it actually might be my favorite part of the theme. But in the main steam menu the white seems to blend with everything instead of making different divisions in the menu look separate and organized. I think if you could make a system that distinguished different spots like the chat log and the chat input box, I would be a lot happier. So if you were to make a dark theme, I would like it being used in the main library section of steam as well.
I hope you do not feel bogged down by our comments but can receive constructive criticism.

**Comment by RealoFoxtrot** [comment]
I know i've been outspoken on the Theme, and would like to restate similar to what i did in steam.

I like the nice bright theme (except for the contrast between white and Steam's Blue)
However, It's TOO white.
I was sent the link to this, and had a look around comparing colour tools https://www.google.com/design/spec/style/color.html#color-color-palette

Material's general "White" is actually grey 50 (RGB all = 250)
While Outright white is White (RGB all = 255)
Using a colour picker, I checked what colour pressure2 is, and it comes up at RGB all = 252

My suggestion for the white would be to drop it down to Grey 100 or 200
(Also, possibly make the Chat Colours a bit darker too..)

Outside of that, the only other problem i personally had was the fact Library was just so different from the other 3 tabs, which all use a shade of blue not found in Material.
The closest i could get was Blue or Blue-Grey 900.

My suggestion for this would be.. well. If you want to stay with material, this won't work, but could you maybe make a Similar background style overlay to how Pressure 1 had with the green? only this time a different colour?

**Comment by Kape98** [comment]
About being most of the websites white/light-themed, I actually use custom dark themes for the websites I visit frequently using the Chrome Extension Stylish.
I really dislike light themes because they are too much for my eyes, how about adding an option to switch from light to dark, a win-win for everybody right?
If you're still unsure about this, I suggest making a poll and announce it on the Steam group, asking if the voter would use a dark theme rather than the light one.
(lol I even have github custom themed http://puu.sh/nLq7T/1dc821defa.png )

**Comment by Nikey646** [comment]
It's simple. I came from Pixel Vision. It was a dark theme. I prefer a darker theme, because it suits every other application i have, which is generalized to be dark, or darker, whether through design or theme.

I didn't come for a theme so bright it could illuminate my entire room if i put it on my main monitor. Atm, i'm using the community made dark version, until you release a complete dark overhaul, or bring Classic further up to date (Since i encountered some issues on the version i was using).

I personally think it was unfair to just suddenly change "Pressure" from the previous design to google's (imo, horrible) material design out of no where (afaik), and tagging a "²" onto the end of it. Not everyone is a fan of the material design concept (Soooo much wasted space imo), and switching it from a general dark to mostly light theme is even crueler.

If this was in an environment where money was to be made, such a change would be almost unacceptable unless the community for the product demanded the change, and while looking at the steam discussions for the Pressure group i can see some people like the change (Each to their own, of course!) i don't see anything requesting that Pressure go the way of Material Design.

Also, for your argument of 

> "Well, the majority of applications and websites are white or light themed, why's this any different?"

I can surely say that any website, or application, that offers a dark theme, will promptly have me changing to it. If a website does not offer a dark theme, than i will check to see if there is a UserScript or UserStyle that fixes this problem. Finally if neither of those are available, i will consider finding and using alternative websites, that provide any of those options and abandoning the previous one, unless there is no alternative that provides the same or similar functionality. Does Pressure² provide unique functionality that isn't found anywhere else? Pretty sure it cannot since it's just a generic steam theme that uses the Material Design.

Put simply, what i'm trying to say is; You should provide a dark theme (Even if it's only a different color palette that we need to manually drop in, using Material Design's darker colors), if you refuse to do that, provide support for the original support of Pressure, and if you refuse to do either of those,  risk losing some (Even if it's only me) of your theme's users. It's not a matter of functionality but preference.

Finally, _do not_ try to mix and match Dark/Light color schemes like your "compromise", it just looks bad.

P.S: This comment is meant to be civil, and just my opinion on the matter. If you interpret this as anything other than that, I'm sorry in advance?

**Comment by Asphira** [comment]
Although I know this is a civil response, Pressure² was not particularly meant to be a sudden thing, I know Dirt had been working on it for quite some time more so for himself and had been updating me on its progress, it was kept hush hush initially since it was in very early phases until it was ready to showcase. As any good creation though it was then also released to the community but the outcry definitely was a bit of a shock. Reminds me of when steam went from its old visual style to the 2010 beta, initially people were outraged and annoyed and then it became standard and familiar.

It also should be considered as a completely different skin to the original pressure when being compared as well because it was created from the ground up from what I saw in the development stages, and had a lot more work put into it ensuring as many elements as possible were actually themed then the original pressure. I found the theme growing on me personally pretty quickly after using it for a while when bug fixing or bug hunting, even over my own personal creation Blue Pulse, (granted I flick back to it oftenly to try and update it or use it for variety).

It has however has been noticed that a good collection of Pressures original fan base would rather a darker version, one of which I am sure will come to fruition eventually, I do not forsee much work from Dirt or myself towards Pressure the original itself though, as it has been mentioned before that a darker version would be a new creation most likely (named Pressure Classic) that would take inspiration from Pressure and newer less hacky methods of coding for the intended result (believe me, Pressure 1 is a mess in this department that limits what you can do to it when updating). That and the option of just offering a darker colour palette for Pressure 2 (in a tweakable/swappable method) is a lot more work then it should be, which is why a new skin warrants it more.

tl;dr a dark version will eventually crop up, try and look at pressure 2 as its own entity rather then comparing it to the hacky pressure 1 that is a very different visual style.

**Comment by Nikey646** [comment]
@Asphira I have a few counter arguments i would like to make related to your comment. It also appear by the start of your comment it seems that i came across as a bit aggressive? Sorry :P, it may occur again ;)

While it may not have been intended to be sudden, by keeping it "hush hush", as you put it, you made it a sudden thing. Announcing a redesign from scratch using Material Design, while not providing any information would have changed that. It would have given the community time to react, and even potentially put their opinion on such a change. While i can understand that Skin designers, design skins for themselves first and foremost, if it was planned as a sequel to Pressure from the start this isn't really fair to the users.

As for the it being "considered a completely different skin", you expect people to care what you expect it to be considered? It has the same name, with a cheeky ² thrown onto the end of it. If it was meant to be considered a different skin, than you should have used a different name, such as "Material Pressure" or something that clearly indicates this is not a direct sequel of the original Pressure skin, or even scrapping the name "Pressure" entirely (I can understand not wanting to do this though, which is why it was listed second.)

For the "Community Outcry", logically expecting otherwise would be the definition of insanity. While i never used steam properly before it changed to the current vanilla skin, I can see the old versions from screenshots and state that the change was not as drastic as the changes that were made in this case.

As for a dark variation, I suspect if you continue with the plan of making the "Dark version" of Pressure² basically be "Pressure Classic" the outcry will likely return, especially if it's meant to use Material Design, as Pressure² does. How is that classic at all? If you mean that it'll be a dark skin similar to the original Pressure than ignore this paragraph. (Side note: It would be better just to provide a Dark version of Pressure², rather than having to maintain two entirely different skins. People will get used to the Material Design, or find an alternative)

Finally, for your tl;dr, refer to the second paragraph about labeling the skin and what people expect from a skin named in fashion of a sequel. Not a very fair thing to expect of people imo.

Slightly long tl;dr: This was done the wrong way, it resulted in miss understandings, complaints, and unhappy community members. It could have, and should have been done better (and arguably more professionally, but it's not fair to expect skin devs to work in a professional manner), whether it was an announcement, or a better choice in naming, or even something else.

P.S: As per my last comment, this comment is meant to be entirely civil, if you read it otherwise, sorry?
P.S.S: Is this comment longer than my original? lolwut?

**Comment by ghost** [comment]
I find darker themes better in general, especially for night time usage. It's the primary reason why I prefer using the original Pressure skin compared to 2 because 2, simply put hurts my eyes in a lot scenarios.

**Comment by FlamingFox911** [comment]
I also agree with @sidspyker. Given exposure to bright backgrounds, my eyes will eventually get tired (as it uses all 3 colour receptors in the eyes), forcing me to either change programs or shut my eyes for some time, essentially doing nothing. Dark themes make it easier on the eyes, allowing me to focus more on what is happening with the program, whether it is typing an essay or reading the news. This is one of the reasons I like to stick with themes like PixelVision and Pressure.

Side Note: Have you considered a config file to set a primary and secondary colour? I'd think it would be a great idea if users of Pressure² could personalize the interface by adjusting the colour schemes. So instead of White and Blue (with Black text), you could do Black and Orange (with White text) for example.

**Comment by ZoneSix** [comment]
Dark themes should be a standard now days. They are much easier on the eyes when using an application/website with a dark theme. Pressure is a dark themed skin, and I very much like it for that. Pressure² shouldn't be an exception. Especially since it's a newer theme. We as developers should start adapting dark themes as a standard and provide both versions. This is beneficial to everyone who wants to use the application/webpage in low light. And also to some screens out there. On some screens it reduces the power draw, and on AMOLED displays... Black colors simply turn the LED off and saves even more power.

I wouldn't be surprised if majority of users use the steam application in low light. To me this is important as I like to always have my steam chats open on my second monitor. And I don't always wanna have a bright window shining at my face distracting my eyes from what I am doing on my main monitor. But I also don't want to give up on the easy chat feed access. I like the minimalistic design of Pressure², and I think it has potential if it receives a dark variant.  But I need the darkness. So for now I'm sticking with the original Pressure skin.

**Comment by noodlecollie** [comment]
I guess I'll leave an opinion on this - the styling of everything else on P2 is gorgeous, but like some others the pure white vs. dark grey contrast everywhere is enough to prevent me from using it. Not only is it harsh on my eyes, but it just doesn't seem consistent with the rest of the theming - white backgrounds seem OK to me for light grey themes, but when it's combined with the dark top frame it just seems a bit like "Woah, suddenly someone's shining a torch in my face from the middle of this window." The attention it demanded from my visual system made me completely miss the fact that my friends list was at the bottom right of the screen, because at a glance the bottom grey bar just looked like an extension of my monitor frame instead of being part of the UI.*

Everything else about the theme is wonderful, but I just can't get over the brightness clash, which the dark frame exaggerates.

(\* I should note I'm on Ubuntu, so my taskbar is up the left hand side.)

**Comment by screwyluie** [comment]
Well as the author? tweaker? whatever you want to call me of the Dark Pressure 2 skin I dislike white. The excuse that "well other things are white, so what's the big deal?" is unsound logic.

People using the original pressure skin were using  it because it was clean and dark. You changed one of the core reasons for use, and made it light. That's fine, it's yours, whatever... but you can't expect your fans to be OK with this change when you've completely disregarded their reason for using the first skin.

The only compromise that you'll find everyone in agreement with is having an optional light/dark. Changing the in game overlay isn't going to win anyone over. Your loyal users want a dark skin, like the first one. Your new users who were attracted by the white theme obviously want a light theme. You can either cater to both or you can ignore one or the other. There is no compromise to be made.

At this point I'm not sure why you don't do both, I've done most of the work for you. I'm sure it's sloppy and not how you would do it, but the end result is there. If you cleaned up my mess and made the two more code compatible, maintaining both would be rather easy. I mean even with my mess it still doesn't take but 20 mins to incorporate your changes to my version manually.

To use you circular logic, if I could turn off white backgrounds on all websites and applications without screwing up the color scheme (eg: black text on a black back ground... it needs to change to white or something) I would. I don't have that option so I can't. It's not that we're OK with it, it's a lack of choice.

**Comment by Js41637** [comment]
Well done on making your version of Pressure Dark unfortunately I don't think Dirt has the motivation to do anything with Pressure anymore. He put a lot of work into designing the original with mockups and shit loads of testing and I think he has very little motivation to want to put effort into changing it to a dark version and making it meet his own standards of quality after the reception and shit he got with this one.

**Comment by DFNCTSC** [comment]
I like how you just stopped posting updates for the skin, after everyone said they preferred it dark. 👊

**Comment by orels1** [comment]
Just wanted to comment with my own reasoning.

I really like the theme. From a design standpoint it's great. But the dark theme is really good for your eyes in the night. That's why Discord has dark theme, and custom css for github or youtube to make it dark - are so popular too, or BetterTTV for Twitch is promoted as a "dark theme switcher" so much. And the list goes on.

There is a huge amount of work put into this skin, and I can totally see how it's frustrating to bend under the end-users will, but I will say basically one thing.

People got used to the old pressure, and it was dark and had a really high contrast which really catches your eye. The bright version can't do that, and I feel like that's where a lot of complaints are coming from. People are used to one thing, and switching to another is just really hard.

Also, the Dark is almost similar to "modern" and "fancy", when bright is associated with old and bland. That's not just my assumption, but the thing that I hear from everywhere.

Anyway. I'm not really expecting you to continue working on this after all the above, but I really want you to see user's point in not-so-shitty-as-some-of-the-comments form.

Good luck! :)

**Comment by Js41637** [comment]
@DFNCTSC Actually there are some commits in the development branch :wink:

**Comment by Kobi-Blade** [comment]
I wonder if the Skin was dropped, development and updates seems to have seized since the Dark Theme discussion started.

**Comment by Skribbles** [comment]
Honestly, yeah, i think a dark theme would be great. It should also use the crystal-like pattern in it, being less of a dark theme and more of a throwback to the original Pressure. The only thing i don't like about Pressure2 is the white theme- This would surely fix that.

**Comment by xcava86x** [comment]
Elaborate on why... I don't like bright things, they hurt my eyes, I prefer dark-themed websites and skins, tend to avoid anything bright.
How more can I elaborate... This is pretty much it. :-\

**Comment by FlaminSarge** [comment]
I believe it's generally accepted by now that a Material Design skin for Steam with a dark motif is something that should exist; because DirtDiglett has halted development on Pressure2 for the moment, I don't believe adding to this conversation is going to go anywhere.

Js41637 is focusing on bugfixes for the bright version of Pressure2 at the moment, and as such is in no position to begin work on a dark version. One person can only do so much.

I believe progress on a dark version can only start when Pressure2's dev version reaches a bug-free and mostly complete state; it wouldn't do to have to change both the Dark and Bright versions every time something needs to be changed.

**Comment by Hank-Riker** [comment]
DirtDiglett,
Thank you for the Pressure² skin. I've used a number of skins over the years, and it's probably the one I've enjoyed the most. I admit it, though. I use dark themes on Win10, and every application that has them, including the dark-tweaked Pressure². I for one would be in favor of it being the default, but to each their own. Just wanted to say thank you for making the skin.
